awx/ui/client/src/templates/job_templates/job-template.form.js

```diff
@@ -43,6 +43,19 @@ function(NotificationsList, i18n) {
                     column: 1,
                     ngDisabled: '!(job_template_obj.summary_fields.user_capabilities.edit || canAddJobTemplate)'
                 },
+                organization: {
+                    label: i18n._('Organization'),
+                    type: 'lookup',
+                    list: 'OrganizationList',
+                    sourceModel: 'organization',
+                    basePath: 'organizations',
+                    sourceField: 'name',
+                    dataTitle: i18n._('Organization'),
+                    dataContainer: 'body',
+                    dataPlacement: 'right',
+                    ngDisabled: '!(job_template_obj.summary_fields.user_capabilities.edit || canAddJobTemplate)',
```

Comment:
Given that it is supposed to be read-only and implicitly inherited from the project relationship, I'm wondering if this field should _always_ be disabled.

Reply:
I think we just revert this file entirely. It was from the previous version of the feature when it was still editable. I don't think the input box is desired at all.

Reply:
sgtm :+1: